test/test_create_observations.js

```diff
@@ -0,0 +1,65 @@
+import app           from './server';
+import supertest     from 'supertest';
+import should        from 'should';
+import { createObservations } from '../src/constants';
+import { CreateObservationsRequestOneObservation,
+         CreateObservationsRequestMissingDatastream,
+         CreateObservationsRequestMissingPhenomenonTime,
+         CreateObservationsRequestComponentsDataArrayMismatch,
+         CreateObservationsRequestNotArray } from './constants';
+
+const endpoint = '/v1.0/' + createObservations;
+const server   = supertest.agent(app.listen(8889));
+
+const postSuccess = (body, done) => {
+  server.post(endpoint).send(body)
+  .expect(201)
+  .end((err, res) => {
+    should.not.exist(err);
+    res.status.should.be.equal(201);
```

Comment:
Can we check here that in the body we have a list of the Observations urls that we are expecting?